app/models/state_changer/standard_plate.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module StateChanger
+  # Handles the basic transitions of a standard plate
+  class StandardPlate < StateChanger::Base
+    # Updates the state of the labware to the target state.  The basic implementation does this by updating
+    # all of the TransferRequest instances to the state specified.  If contents is blank then the change is assumed to
```

Comment:
```suggestion
    # all of the TransferRequest instances to the state specified.  If 'contents' is blank then the change is assumed to
```

As suggested elsewhere